include/v810_opt.h

```diff
@@ -62,6 +62,8 @@ typedef struct {
     void (*func)(int, int, int); //pointer to handler func =)
 } operation;
 
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wincompatible-pointer-types"
```

Comment:
GCC doesn't like it that `ins_err` is defined as taking two variables but the type of operation takes a function pointer that takes three. This implicit casting seems like it makes the operation functions a lot more pleasant to work on, so just setting up the ignore.